index.html

```diff
@@ -441,12 +436,44 @@ <h2>Profiling Mechanism</h2>
       ...
     }
     </pre>
+    <p>
+      <span class="rfc2119-assertion" id="profiling-mechanism-5">
+        All Things and Consumers conforming to a profile MUST satisfy the assertions specified in the WoT Thing 
+        Description specification [[wot-thing-description11]],
+        except for the assertion with text &quot;TD 1.1 consumers MUST accept
+        TDs satisfying the W3C WoT Thing Description 1.0 [wot-thing-description] specification.&quot;
+      </span>
+    </p>
+    <p>
+      <span class="rfc2119-assertion" id="profiling-mechanism-6">
+        A profile SHOULD NOT redefine defaults from protocol binding templates [[wot-binding-templates]].
+      </span>
+      E.g. If a profile uses an HTTP protocol binding and the HTTP Binding Template specifies that in an 
```

Comment:
```suggestion
      For instance, if a profile uses an HTTP protocol binding, and the HTTP Binding Template specifies that in an 
```

Reply:
@TallTed Is there a particular reason for this proposed change?

Reply:
While "e.g." (and "i.e.") are *permitted* as sentence starters, I've always been taught not to do so, and they rub me the wrong way. If you feel strongly the other way, I'll live with it.